File: src/main/java/ru/pashkovske/buratino/tinkoff/service/order/OrderService.java
package ru.pashkovske.buratino.tinkoff.service.order;

import ru.pashkovske.buratino.tinkoff.service.model.InstrumentHolder;

public interface OrderService {
    void limitSellBestByOrderBook(InstrumentHolder instrument);
    void limitBuyBestByOrderBook(InstrumentHolder instrument);
}

File: src/main/kotlin/ru/pashkovske/buratino/order/service/OrderService.kt
package ru.pashkovske.buratino.order.service

import ru.pashkovske.buratino.order.model.Order
import ru.pashkovske.buratino.order.model.limit.LimitOrderRequest

interface OrderService {

    fun refreshOrders()
    fun createOrder(orderRequest: LimitOrderRequest): Order
    fun replaceOrder(
        orderId: String,
        newOrderRequest: LimitOrderRequest
    ): Order
    fun refreshOrder(orderId: String): Order
    fun cancelOrder(orderId: String): Order
    fun isOrderCompleted(orderId: String): Boolean
}